src/olympia/devhub/views.py

```diff
@@ -1939,6 +1939,7 @@ def onboarding_subscription_success(request, addon_id, addon):
             payment_completed_at=datetime.datetime.now(),
         )
         log.info('PromotedSubscription %s has been completed.', sub.pk)
+        sub.promoted_addon.approve_for_addon()
```

Comment:
I guess we need to update the content of the template (`onboarding_subscription.html`) too.

Reply:
Oh, I didn't read the text.  Yep.

Reply:
I think you'll have to rebase your patch too

Reply:
indeed, and `addon_already_promoted` needed taking into account now that's a thing https://github.com/mozilla/addons-server/pull/15822/commits/2fed0781911f2da5ede75bc9268215dad9461bc6